reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -622,7 +652,11 @@ public Emission getReason() {
 		 * @deprecated to be removed shortly after 3.4.0-RC1. Use {@link #tryEmitComplete()} and handle the result.
 		 */
 		@Deprecated
-		void emitComplete();
+		default void emitComplete() {
+			emitComplete(EmitFailureHandler.FAIL_FAST);
+		}
+
+		void emitComplete(EmitFailureHandler failureHandler);
```

Comment:
missing javadoc. insist on the fact that this should rely on `tryEmitComplete` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496589276)